core/project_api.php

```diff
@@ -679,9 +679,14 @@ function project_get_all_user_rows( $p_project_id = ALL_PROJECTS, $p_access_leve
 		db_param_push();
 		$t_query = 'SELECT id, username, realname, access_level
 				FROM {user}
-				WHERE enabled = ' . db_param() . '
-					AND access_level ' . $t_global_access_clause;
-		$t_result = db_query( $t_query, array( $t_on ) );
+				WHERE access_level ' . $t_global_access_clause;
+
+		if ( !$p_include_disabled_users ) {
+			$t_query .= ' AND enabled = ' . db_param();
+			$t_result = db_query( $t_query, Array( $t_on ) );
```

Comment:
This query may have, or not, parameters based onthe value of $p_include_disabled_users.
For correctness, when the query does not have parameters, the parameter array is not used in db_query(), but it's also important to **not** issue db_param_push(). Doing so has historically causes problems in postgres... iirc.

I think that after the latest db_api rewrite, that's is no longer an issue, as db_query() executes a wrapper for a DbQuery object. BUT... in code like this, i usually rewrite the query in the new syntax, just in case...